modin/core/execution/utils.py

```diff
@@ -29,3 +31,62 @@ def set_env(**environ):
     finally:
         os.environ.clear()
         os.environ.update(old_environ)
+
+
+if "_MODIN_DOC_CHECKER_" in os.environ:
+
+    # The doc checker should get the non-processed functions
```

Comment:
why?

Reply:
Because doc checker receives not a function but ObjectRef instead and fails.